SOURCES/src/kernel/MAKEUTIL.MAK

```diff
@@ -465,14 +473,13 @@ moxcppls.obj :  moxcppls.asm  page.inc  moxcpdat.inc  mosscbdf.inc \
 #
 #  Utility programs written in C.  (And Pascal, temporarily)
 #
-compfile.exe: compfile.c
-        cl -c compfile.c
-        link compfile/e,,,rsasmall
+compfile.exe: ..\mos5src\compfile.c
+        cl -c -Ic:\include ..\mos5src\compfile.c

```

Comment:
Try to avoid using drive letters, people won't have the same drive setup as you. I fixed a different way see my patch series https://github.com/andrewbird/pcmos386/commits/build02 . Just waiting for your PR to go in before I rebase off master.

Reply:
Retracted.  Same as above.